web/src/sections/sidebar/SidebarBody.tsx

```diff
@@ -7,17 +7,21 @@ export interface SidebarBodyProps {
   actionButton?: React.ReactNode;
   children?: React.ReactNode;
   footer?: React.ReactNode;
+  scrollKey: string;
```

Comment:
This one can stay non-optional. `SidebarBody` can enforce that it itself is scroll-save-able, and thus definitely requires this parameter to be passed into it.